deployment/pipecat-cloud/rest-reference/endpoint/stop.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: "Stop an Agent session"
+description: "Stop a running agent session and clean up its resources."
+openapi: "DELETE /{agentName}/sessions/{sessionId}"
```

Comment:
These should be an `/agents/` path segment included.  The full path is `/v1/agents/{agentName}/sessions/{sessionId}`.